src/test/java/seedu/address/testutil/ParentBuilder.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalStudents.ALICE;
+
+import seedu.address.model.person.Parent;
+import seedu.address.model.person.Student;
+
+/**
+ * A utility class to help with building Parent objects.
+ */
+public class ParentBuilder extends PersonBuilder {
+
+    public static final Student DEFAULT_CHILD = ALICE;
+
+    private Student child;
+
+    /**
+     * Creates a {@code ParentBuilder} with the default details.
+     */
+    public ParentBuilder() {
+        super();
+        child = DEFAULT_CHILD;
+    }
+
+    /**
+     * Initializes the ParentBuilder with the data of {@code parentToCopy}.
+     */
+    public ParentBuilder(Parent parentToCopy) {
+        super(parentToCopy);
+        child = parentToCopy.getChild();
+    }
+
+    /**
+     * Sets the {@code child} of the {@code Parent} that we are building.
+     */
```

Comment:
I like how the JavaDoc comments are clear and concise, and have explicit links to the relevant classes!